styles/Links.module.css

```diff
@@ -87,7 +87,7 @@
   padding-top: 18px !important;
 }
 
-.priority::before {
+.official::before,.unofficial::before,.priority::before{
   content: 'Our sites';
   color: var(--background-dark);
   background-color: #6C479A;
```

Comment:
We are using it to pin our sites at top. What's the use for other sites to be inside this ?